### ISSUE TITLE
using sudo for elevation needs root use logged in to resin too

### ISSUE THREAD

**Issue by imrehg** [resin-io/resin-cli#385]
The  `config read`, `config write`, `config reconfigure` group of instructions need elevation to work with the SD cards, and thus have to run `sudo resin config <etc>`. But this means that the root user will need to log in to the same resin account as well, and any other `user 2` on the same machine with root access can access the `user 1`'s resin account because of that.

Instead of running with sudo, should probably try to elevate internally when needed? 

Another possibility is not requiring resin login for operations that are just local (looking at them, all three just needs data from the card, not from the servers). This would allow running with sudo but not logging in ot resin.

**Comment by jviotti** [comment]
I agree. Let's make most `config` commands available without requiring to log in.

**Comment by pdcastro** [comment]
Related to #1443

**Comment by pdcastro** [comment]
A potential mitigatory implementation (a.k.a. hack?) is described in #550 (see my comment there).
